packages/next/src/utils/config.ts

```diff
@@ -56,6 +56,11 @@ export function createWebpackConfig(
         test: /\.tsx?$/,
         use: [defaultLoaders.babel],
       },
+      {
+        test: /\.js?$/,
```

Comment:
Can we update the existing loader to test on `/\.(j|t)sx?$/` instead? This will capture `.ts,.tsx,.js,.jsx`

Reply:
Updated at f96d85f.